lib/security_response_header_filter.js

```diff
@@ -59,7 +59,8 @@ module.exports = function addCSPHeaders(hyper, req, next, options) {
         // Set up basic CORS headers
         rh['access-control-allow-origin'] = '*';
         rh['access-control-allow-methods'] = 'GET';
-        rh['access-control-allow-headers'] = 'accept, content-type';
+        rh['access-control-allow-headers'] =
+            'accept, accept-encoding, origin, accept-language, content-type';
```

Comment:
@gwicke Safari 9 sends it with an OPTIONS request. As I understand, normally you need to send back all the headers from OPTIONS preflight request to allow it and I couldn't find anything about `Origin` header being somehow specific